apps/asap-cli/src/scripts/algolia/delete-index.ts

```diff
@@ -0,0 +1,31 @@
+/* istanbul ignore file */
+import algoliasearch from 'algoliasearch';
+
+export type DeleteAlgoliaIndex = {
+  algoliaAppId: string;
+  algoliaCiApiKey: string;
+  indexName: string;
+};
+
+export const deleteAlgoliaIndex = async ({
+  algoliaAppId,
+  algoliaCiApiKey,
+  indexName,
+}: DeleteAlgoliaIndex): Promise<void> => {
+  const client = algoliasearch(algoliaAppId, algoliaCiApiKey);
+  const unlinkIndex = async (name: string) => {
+    const index = client.initIndex(name);
+    return index.setSettings({ replicas: [] }).wait();
+  };
+  const indices = await client.listIndices();
+  indices.items
+    .filter(({ name }) => name.startsWith(indexName))
+    .forEach(async ({ name }) => {
```

Comment:
Piotr: Having the comment above in mind I think we should simply get the primary index in question, get all of its replicas, the unlike them all and remove each one?